docker/Dockerfile

```diff
@@ -212,6 +230,7 @@ RUN install-from-source https://gitlab.gnome.org/GNOME/glib/-/archive/2.63.3/gli
     -Ddefault_library=both \
     -Dinternal_pcre=true \
     && cp /usr/local/lib/glib-2.0/include/glibconfig.h /usr/local/include/glibconfig.h
+RUN install-package libusb
```

Comment:
Why are we installing a package for this rather than building it.
In general, we would only want to install a package if it's used TO build something. Not as a dependency of a library itself.